net/natmap/Makefile

```
@@ -0,0 +1,38 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=natmap
+PKG_VERSION:=20221111
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/heiher/natmap.git
+PKG_SOURCE_DATE:=2022-11-11
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_MIRROR_HASH:=4359bdddd16d7f97af4ee03f814ea4e649554c30eeed93e5f4fc586151d2476e
+
+PKG_MAINTAINER:=Richard Yu <yurichard3839@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=License
+
+PKG_USE_MIPS16:=0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/$(PKG_NAME)
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=TCP/UDP port mapping for full cone NAT, any host can access internal services of private address broadband
+	MAINTAINER:=heiher
+	URL:=https://github.com/heiher/natmap
```

```suggestion
define Package/natmap
  SECTION:=net
  CATEGORY:=Network
  TITLE:=TCP/UDP port mapping for full cone NAT, any host can access internal services of private address broadband
  URL:=https://github.com/heiher/natmap
```

- Do not use $(PKG_NAME), as it reduces readability.
- Use two `<space>` instead of `<tab>`.
- `MAINTAINER` can be removed, this is actually meant `PKG_MAINTAINER`.